doc/source/whatsnew/v0.25.1.rst

```diff
@@ -54,7 +54,7 @@ Numeric
 ^^^^^^^
 - Bug in :meth:`Series.interpolate` when using a timezone aware :class:`DatetimeIndex` (:issue:`27548`)
 - Bug when printing negative floating point complex numbers would raise an ``IndexError`` (:issue:`27484`)
--
+- Bug where :class:`DataFrame` arithmetic operators such as :meth:`DataFrame.mul` with a :class:`Series` with axis=1 would raise an ``AttributeError`` on large dataframes with numexpr installed (:issue:`27636`)
```

Comment:
What determines what a `large data frame` is? I think clarifying that would be helpful

Reply:
I have added the detail on why a large DataFrame caused this. I am reluctant to quote the exact location of the threshold as its not publicly inspect-able or settable. I think the key point here is that if you pass the first check that the computation is likely to be large enough to bother invoking numexpr then that is when the bug arises.

Reply:
I think this is fine.